tox.ini

```diff
@@ -24,11 +24,35 @@ deps =
 commands =
     towncrier {posargs}
 
-[testenv:docs]
+[docs]
+change_dir = docs
 extras =
     docs
+# -v: verbose logging
+# -W: make warnings into errors
+# --keep-going: continue on errors
+# -E: force rebuild of environment
+# -a: read/parse all files
+# -n: nitpick mode
+sphinx_verbose_args = -v -W --keep-going -E -a -n
+
+[testenv:docs]
+change_dir = {[docs]change_dir}
+extras = {[docs]extras}
+commands =
+    sphinx-build -W -b html . _build/html
+
+[testenv:docs-links]
```

Comment:
`docs-links`  is *currently* just a link check, but I can imagine other checks (most notably, a spellchecker), so `docs-lint` is probably a better name

Reply:
Updated.